tools/cpp/windows_cc_configure.bzl

```diff
@@ -269,28 +274,47 @@ def setup_vc_env_vars(repository_ctx, vc_path):
         env_map[key] = escape_string(value.replace("\\", "\\\\"))
     return env_map
 
+def _get_latest_subversion(repository_ctx, vc_path):
+    """Get the latest subversion of a VS 2017/2019 installation.
+
+    For VS 2017 & 2019, there could be multiple versions of VC build tools.
+    The directories are like:
+      <vc_path>\\Tools\\MSVC\\14.10.24930\\bin\\HostX64\\x64
+      <vc_path>\\Tools\\MSVC\\14.16.27023\\bin\\HostX64\\x64
+    This function should return 14.16.27023 in this case."""
+    if not _is_vs_2017_or_2019(vc_path):
+        auto_configure_fail(repository_ctx, "Cannot calculate latest subversion from VC build tool older than 2017")
```

Comment:
What does this error mean for the user, what can they do to avoid it? Please rephrase the error message to make it actionable for the user.

Reply:
Actually, this error message will never be thrown, because we always check the VC version before invoking this method. So I'll just remove it.

Reply:
Actually, this error message will never be thrown, because we always check the VC version before invoking this method. So I'll just remove it.